packages/p2p-v2/src/components/BuySellForm/BuySellAmount/BuySellAmount.scss

```diff
@@ -8,5 +8,9 @@
             flex-direction: column;
             gap: 1.6rem;
         }
+
+        .deriv-input__field {
+            font-size: 1.2rem;
```

Comment:
is this not being handled by `Input` component?

Reply:
i had to set it to a specific size cause you can't pass the text size for the input